content/06.discussion-a.md

```diff
@@ -4,59 +4,63 @@
-Managing full research projects or laboratories on GitHub requires careful thought as to how to delegate tasks such as reviewing pull requests or creating issues.
-For example, collaboratively writing a paper using GitHub, as we have done here, may involve a learning curve for co-authors less familiar with the intricacies of GitHub, and requires overhead to set up the repository using GitHub Actions.
-Despite the many potential applications of GitHub to EEB research, we acknowledge that there will still be many times when researchers might look to other platforms for research collaboration.
+Most of these uses can be quickly integrated into the research practices of users (_e.g._, storing data, creating virtual notebooks, making code citable).
+Making stored data and code citable usually involves creating a repository on GitHub, pushing code and data, and then integrating a DOI minting service to the repository (_e.g._, with Zenodo; see below).
+On the other hand, some examples we described here, including course material development, web hosting, and delivery, require greater effort and time commitment, but have the potential to make EEB research more open, accessible, and collaborative.
+Managing full research projects or laboratories on GitHub require careful thought as to how to delegate tasks such as reviewing pull requests or creating issues.
+For example, collaboratively writing a paper using GitHub, as we have done here, involves a learning curve for co-authors less familiar with the intricacies of GitHub, requires overhead to set up the repository using GitHub Actions and the commitment from all co-authors to use GitHub when modifying and reviewing the text.
+Despite the many potential applications of GitHub to EEB research, we acknowledge that researchers might still look to other platforms for research collaboration.
 
 ### Other platforms for collaboration
 
 <!--*Contributors to this section: Rob*-->
 
 Despite its strong collaborative potential, we describe two use cases where GitHub's features fall short of highly collaborative work emblematic of EEB research.
 
+
 First, real-time document editing is still best performed on other platforms (_e.g._, cloud-stored documents from Microsoft Word, Google Docs, hackMD (<https://hackmd.io/>)).
-Second, operations that are dependent on other software might not be easily accomplished in GitHub, such as manipulating figures or tables.
-Although creating tables and figures can be achieved through code, users may choose other software to collaboratively brainstorm figures and tables (_e.g._ Google Slides, Google Sheets).
+Second, operations that are dependent on other software might not be easily achievable in GitHub, such as manipulating figures or tables.
+Although creating tables and figures can be achieve through code, users may choose other software to collaboratively brainstorm figures and tables (_e.g_. Google Slides, Google Sheets; but see GitHub Discussions).
```

Comment:
With the changes you made here, we now use "achievable" and "achieve" so I suggest either reverting back to previous wording to avoid repetition or use another synonym for achieve.

Reply:
No problem. In the sentence below, should it read "can be achieved"?

Reply:
Will update text, thanks!